docs/data/sdks/sdk-comparison.md

```diff
@@ -0,0 +1,113 @@
+
+unsetParamsReferrerOnNewSession
+
+#### Workflow 
+
+|                 Web Attribution V1                     |                Web Attribution V2                     |Maintenance Web Attribution|
+|:--------------------------------------------------------------------------------------------------------------:| :--------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------:|
+| ![Web Attribution V1](../../assets/images/sdk/web-attribution-v1.drawio.svg)  | ![Web Attribution V2](../../assets/images/sdk/web-attribution-v2.drawio.svg)  | ![Maintenance Web Attribution](../../assets/images/sdk/maintenance-web-attribution.drawio.svg)  |
+
+#### Archtecture
+
+| <div class="big-column">  Web Attribution V1 </div> | <div class="big-column"> Web Attribution V2 </div> | <div class="big-column">  Maintenance Web Attribution </div>|
+| --- | --- | --- |
+| Implemented through `webAttributionPlugin` Plugin. The behavior will be the same if you set `config.trackNewCampaigns = false`.  | Implemented through `webAttributionPlugin` Plugin. | Build in logic. |
+
+## Android
+
+| <div class="big-column">Feature</div> | [Android Kotlin](../android-kotlin/) | [Android Android](../android/) |
+| --- | --- | --- |
+| Package | [com.amplitude:analytics-android](https://mvnrepository.com/artifact/com.amplitude/analytics-android) | [com.amplitude:android-sdk](https://mvnrepository.com/artifact/com.amplitude/android-sdk) |
+| SSL Pinning | TBD | Supported. Check [here](../android/#ssl-pinning) for the setup. |
+| Configuration | Configuration is implemented by the configuration object. Configurations need to be passed into Amplitude Object during initialization. [More configurations](../android-kotlin/#configuration). | Support explicity setter methods. [More configurations](../android/#configuration). |
+| Logger provider | ConsoleLoggerProvider() by default. Fully customizable. | Amplitude Logger. Not customizable. |
+| Storage Provider | InMemoryStorageProvider() by default. File storage. Fully customizable. | SQLite Database. |       
+| Customization | Plugins | Middelware |
+| Server Endpoint | HTTP V2 API | HTTP V1 API |
+| Batch API| Supported, with configuration. | Not supported.|
+
+## iOS
+
+| <div class="big-column">Feature</div> | [Latest iOS SDK](../ios/) | [Maintenance iOS SDK](../ios-swift/) |
+| --- | --- | --- |
+| Package | AmplitudeSwift | [Amplitude](https://cocoapods.org/pods/Amplitude-iOS) |
```

Comment:
Add link for Swift package?

Reply:
The AmplitudeSwift link in Cocoapod has some issues. And I feel that's the issue of cocoapod. We also mentioned this in other ticket before.